src/cargo/util/lints.rs

```diff
@@ -916,4 +918,78 @@ mod tests {
         let actual = format!("{:#?}", actual);
         snapbox::assert_data_eq!(actual, expected);
     }
+
+    #[test]
+    fn ensure_updated_lints() {
+        let path = file!();
```

Comment:
nit: unlikely to be a problem for `cargo` but `file!` doesn't always work which is what `snapbox::utils::current_rs!()` is for